### PR TITLE
Overview

### PR DESCRIPTION
**Pull request by CjS77**
Summary of IRC discussions from week of 25 October 2018

### CODE REVIEW

File: proposals/181029-overview.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+This suggests the partitioning of the Tari network into
+two layers:
+
+* A base layer that provides a public ledger of base token transactions, secured by proof of work (maximise Security), and
+* A second layer that manages digital asset state that is very fast and cheap, but which can fall back on the base layer's security when required.
+  
+## The Base Layer
+
+The only system that we _know_ currently works as a base layer capable of delivering at least the first three must-have 
+properties above (security, decentralisation, ceonsorship resistance) is a token built on a proof-of-work-based
```
Comment (kngako) [inline proposals/181029-overview.md]: Spelling mistake for `censorship`

Comment (CjS77) [reply proposals/181029-overview.md]: 👍

File: proposals/181029-overview.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+  Given Tari's relationship with Monero, a merge-mined strategy with Monero makes the most sense.
+  
+## The second layer
+
+They key properties of the digital assets network are speed and scalability, without compromising on security. Here, we
+can definitely compromise on decentralisation, for example. In fact, in many ways it's desirable, since the vast majority
+of assets (and their issuers) don't need or want _the entire network_ to validate every state change in their asset
+contracts.
+
+Once this trade-off is accepted, it the goals of speed and scalability that we set out above look far more achievable.
```
Comment (ksloven) [inline proposals/181029-overview.md]: it *makes*/*ensures*(?) the goals of speed.....

Comment (CjS77) [reply proposals/181029-overview.md]: Thanks

**Pull request merged**